.github/workflows/release.yml

```diff
@@ -1,11 +1,13 @@
-name: Publish to NPM
+name: Release
 on:
   push:
-    tags:
-      - "*"
+    branches:
```

Comment:
question: Will `semantic-release` generate, publish, and maintain tags for the repo?

Reply:
Yep!